diff_drive_controller/test/config/test_diff_drive_controller.yaml

```diff
@@ -17,6 +17,7 @@ diff_drive_controller:
     pose_covariance_diagonal: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
     twist_covariance_diagonal: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
 
+    position_feedback: false
```

Comment:
is there other option missing: `position_feedback: true`

Reply:
No, in my previous comments I also observed that this file is not being loaded correctly by the tests. So I just added the line as an example of configuration and reference, but it is not being used by tests.

Reply:
Long story short, we need to address this ticket: https://github.com/ros-controls/ros2_controllers/issues/48

Reply:
@bmagyar is your reply to me or @destogl? To me it seems #48 is not a blocking PR for this one.

Reply:
I was not meaning to block this PR with that comment, just cross referencing